md/data/importer.py

```diff
@@ -91,14 +97,28 @@ def fix_TIME_OF_STOP(s):
 
 
 def compute_AGE(row):
+    dob = row['DOB']
     stop_date = row['date']
-    dob = row['DOB_as_dt']
-    if pd.isnull(dob) or dob > stop_date:
-        logger.info('Bad DOB vs. stop date: "%s", "%s"', dob, stop_date)
-        return 0
+
+    m = DOB_re.match(dob)
+    if not m:
+        age = 0
     else:
+        dob_year = int(m.group(3))
+        stop_date_year = stop_date.year
+        stop_date_year_of_century = stop_date_year % 100
+        stop_date_century = int(math.floor(stop_date_year / 100.0)) * 100
+
+        if dob_year >= stop_date_year_of_century:
+            dob_year += (stop_date_century - 100)
+        else:
+            dob_year += stop_date_century
+
+        dob = datetime.datetime(dob_year, int(m.group(1)), int(m.group(2)))
         delta = stop_date - dob
-        return int(delta.days / 365.25)
+        age = int(delta.days / 365.25)
```

Comment:
Wow those 2 year dates are hairy. Took me a little to figure out what was going on here, but looks like you came up with a good solution!